pom.xml

```diff
@@ -574,7 +574,7 @@
         <plugin>
           <groupId>org.owasp</groupId>
           <artifactId>dependency-check-maven</artifactId>
-          <version>5.2.1</version>
+          <version>5.2.4</version>
```

Comment:
please also cite this change in the commit message and update JIRA case

Reply:
@eolivelli Done